src/server/test/web/readingsCompareMeterQuantity.js

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/*
+    This file tests the readings retrieval API compare chart meters.
+    See: https://github.com/OpenEnergyDashboard/DesignDocs/blob/main/testing/testing.md for information.
+*/
+const { chai, mocha, app } = require('../common');
+const { prepareTest,
+    expectCompareToEqualExpected,
+    getUnitId,
+    METER_ID,
+    unitDatakWh,
+    conversionDatakWh,
+    meterDatakWh } = require('../../util/readingsUtils');
+
+mocha.describe('readings API', () => {
+    mocha.describe('readings test, test if data returned by API is as expected', () => {
+        mocha.describe('for compare charts', () => {
+            mocha.describe('for meters', () => {
+                // Test 15 minutes over all time for flow unit.
+                mocha.it('Expected and actual data should line up', async () => {
+                    await prepareTest(unitDatakWh, conversionDatakWh, meterDatakWh);
+                    // Get the unit ID since the DB could use any value.
+                    const unitId = await getUnitId('kWh');
+                    const expected = [ '7962.23097109771', '8764.06090894387', '2022-10-31 17:00:00', 'P1D' ];
```

Comment:
Sorry for not noticing this earlier but the compare API only returns 2 values (as checked in expectCompareToEqualExpected()). Thus, the last two values in the expected array should not be there.

Reply:
Alright fixed!